hs_core/urls.py

```diff
@@ -31,10 +34,12 @@
     url(r'^resource/(?P<pk>[A-z0-9]+)/files/(?P<filename>[^/]+)/$',
```

Comment:
@alvacouch @pkdash Our download path url allows anything with a url pattern like ```url(r'^download/(?P<path>.*)$', 'django_irods.views.download')```. Our view function does the check and serve files only when requested files are valid. That seems to be simple and works well. In that ```resource/<pk>/files/<filename>``` url for REST endpoint, if filename is only the file name without path, we should probably exclude ```/```, otherwise, I'll vote for non-restrictive url pattern for file names or file path.